spi/max7219_32x8_spi/README.adoc

```diff
@@ -0,0 +1,40 @@
+= Attaching a Max7219 driving an 32x8 LED display via SPI
+
+This example code shows how to interface the Raspberry Pi Pico to a Max7219 driving an 32x8 display. The particular device used is interfaced via SPI at 3.3v. This PCB uses 4 Max7219 devices, each driving an 8x8 LED matrix, giving the total of 32x8. These devices are designed to be cascaded together in this way, and SPI commands are passed down the line of devices as required.
```

Comment:
"32x8 **LED** display" ?

Reply:
LED is correct.

Reply:
Sorry, in hindsight I guess that comment was a bit ambiguous - I was asking if the "driving an 32x8 display" should be changed to "driving an 32x8 LED display". (also, the "an" should probably be changed to "a"?)